pkg/kn/commands/plugin/list.go

```diff
@@ -74,7 +74,7 @@ func listPlugins(cmd *cobra.Command, flags pluginListFlags) error {
 	if flags.verbose {
 		fmt.Fprintf(out, "The following plugins are available, using options:\n")
 		fmt.Fprintf(out, "  - plugins dir: '%s'%s\n", commands.Cfg.PluginsDir, extraLabelIfPathNotExists(pluginPath))
-		fmt.Fprintf(out, "  - lookup plugins in path: '%t'\n", commands.Cfg.LookupPluginsInPath)
+		fmt.Fprintf(out, "  - lookup plugins: '%t'\n", commands.Cfg.LookupPlugins)
```

Comment:
since this shows up if verbose, lets keep `lookup plugins in $PATH`?

Reply:
See @rhuss's comment